### ISSUE TITLE
security update required for lodash dependency

### ISSUE THREAD

**Issue by sido420** [primus/omega-supreme#86]
```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ high          │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.11                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ omega-supreme                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ omega-supreme > async > lodash                               │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/782                       │
```

yarn audit ^^
I have 3.0.0 of omega supreme installed

**Comment by lpinca** [comment]
Try with a fresh reinstall there is no lodash dependency in `async@3.2.0`.

**Comment by lpinca** [comment]
It looks like `omega-supreme@3.0.0` is using `async@2.6.0`. Will cut a new version soon.

**Comment by lpinca** [comment]
It's weird anyway as it should pick a fixed version of `lodash` as per https://github.com/caolan/async/blob/v2.6.2/package.json#L22.

**Comment by lpinca** [comment]
I've just released `omega-supreme@3.0.1`.
